packages/apps/plugins/plugin-status-bar/src/components/StatusBarImpl.tsx

```diff
@@ -0,0 +1,45 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import { DiscordLogo, Lightning, PaperPlaneTilt } from '@phosphor-icons/react';
+import React from 'react';
+
+import { Popover } from '@dxos/react-ui';
+
+import { FeedbackForm } from './FeedbackForm';
+import { StatusBar } from './StatusBar';
+
+export const StatusBarImpl = () => {
```

Comment:
Interestingly I feel like elsewhere I've inverted these name (impl for the internal "pure" version and then plain name for the exported general usage component)

Reply:
Good point. It's tricky when you have components / plugins sharing names. I'm vacillating between both options.